lightning/src/chain/keysinterface.rs

```diff
@@ -449,6 +569,8 @@ impl Readable for InMemoryChannelKeys {
 			InMemoryChannelKeys::make_local_keys(&secp_ctx, &funding_key, &revocation_base_key,
 			                                     &payment_key, &delayed_payment_base_key,
 			                                     &htlc_base_key);
+		let user_id_1 = Readable::read(reader)?;
```

Comment:
the fact that the derivation params are, in fact, user IDs should probably be documented a bit more widely

Reply:
In fact they can be user_id but they can whatever else you want, changed names to `params_1`/`params_2`